test/brsTypes/BrsObjects.test.js

```diff
@@ -0,0 +1,13 @@
+const { AssociativeArray, BrsObjects } = require("../../lib/brsTypes");
+const BrsError = require("../../lib/Error");
+
+describe("BrsObjects", () => {
+    beforeEach(() => BrsError.reset());
+
+    describe("new object instances", () => {
```

Comment:
Neato!  These are great to have.  As we add more components this may become a pain to navigate, but we'll deal with that in the future 🤷‍♂️